mubench.pipeline/tasks/implementations/compile_misuse.py

```diff
@@ -27,13 +26,11 @@ def run(self, misuse: Misuse, version_compile: VersionCompile):
         logger.info("Compiling %s...", misuse)
 
         logger.debug("Copying misuse sources...")
-        CompileMisuseTask._copy_misuse_sources(version_compile.original_sources_path,
-                                               misuse,
-                                               pattern_compile.misuse_source_path)
+        for source_path in version_compile.original_sources_paths:
```

Comment:
It seems we have this logic (finding a file in one of the sources/classes directories) in multiple places. Can we centralize and reuse it?